Search.jsp

```diff
@@ -144,12 +147,13 @@ footer
 <sql:setDataSource user="root" password="Fathihachellam.1" url="jdbc:mysql://localhost:3306/ooad?autoReconnect=true&&useSSL=false" driver="com.mysql.jdbc.Driver" var="db"/>
 
 <sql:query var="result" dataSource="${ db}">
-select * from article where title like '%<%=request.getAttribute("search2") %>%'
+select * from article where title like '%<%=request.getAttribute("search2") %>%' or abstracts like '%<%=request.getAttribute("search2") %>%' or highlight like '%<%=request.getAttribute("search2") %>%' order by time desc

```

Comment:
What does `search2 `mean?  Could you add a comment for explaining `search2`?

**Hui**

Reply:
It's a string that passes the search keyword through HTTP request from the servlet to the web page.

Reply:
> It's a string that passes the search keyword through HTTP request from the servlet to the web page.

Thanks @mjhmohamed .  Could you say that in `Search.jsp`, i.e., make comments for `search2 `?

Reply:
will do.